test/system/030-run.bats

```diff
@@ -1419,17 +1422,30 @@ EOF
         # Any other error is fatal
         die "Cannot create idmap mount: $output"
     fi
+    if findmnt "$romount" >/dev/null ; then
+        die "'podman run' did not umount $romount"
+    fi
 
-    run_podman run --security-opt label=disable --rm --uidmap=0:1000:10000 --rootfs $romount:idmap stat -c %u:%g /bin
+    mkdir -p $PODMAN_TMPDIR/shared-volume
+    run_podman run --security-opt label=disable --rm --uidmap=0:1000:10000 -v $PODMAN_TMPDIR/shared-volume:/a-shared-volume:shared --rootfs $romount:idmap stat -c %u:%g /bin
```

Comment:
you add a shared volume here but don't do actually anything with it, is that right?

Is the only check here that we can mount it shared? And then that the root mount was removed? In this case it might be worth a comment why we add a shared mount.

Reply:
I am adding it only to test the `if isShared {` condition in the `createRootlessContainer()` function